example-get-started/generate.sh

```diff
@@ -170,9 +178,11 @@ echo "plots:
 dvc repro
 git add .gitignore dvc.yaml dvc.lock eval
 tick
-git commit -m "Create evaluation stage"
+git commit -am "Create evaluation stage"
 git tag -a "8-evaluation" -m "Baseline evaluation stage created."
 git tag -a "baseline-experiment" -m "Baseline experiment evaluation"
+gto register dvclive:text-classification --version v1.0.0
```

Comment:
do we use dvclive with a custom path in the eval stage? let's try to reuse it ... I don't think we want to have the `dvclive` dir in this project ... or at least is should be a single dir with everything in it

Reply:
Looks somewhat strange to me to have a model registered in `eval/live` TBH. For this particular case, it be good to write to `dvc.yaml` in the root, e.g. using
```
with Live("") as live:
  ...
```
Wdyt @daavoo?

Reply:
> it be good to write to dvc.yaml in the root, e.g. using

I think this would not work or would work in unexpected ways.

We could update the repo to have a parent  `results` output folder and use `results/train` and `results/eval` in each stage, I think it is what Ivan was suggesting

Reply:
Looks like a potentially substantial update to me (I assume we should move `model.pkl` to `results/train` as well?). I suggest to keep this repo simple. Let's use the approach with `results/...` for the `example-gs-experiments` repo and here simply edit `dvc.yaml` in the root like before by calling DVC API.

Reply:
so unless you're against, I'm reverting the changes.